### DISCUSSION TITLE
新手求帮助！仅开了setu和搜图功能(非R18)，没几天提示被TX风控了😭

### DISCUSSION THREAD

**Discussion by tt9891** [Tsuk1ko/CQ-picfinder-robot#345] [问题求助] [answered]
[2022-07-21 15:51:26] [ERROR]: Protocol -> sendPacket msg error: 46
[2022-07-21 15:51:32] [WARNING]: 群消息发送失败: 账号可能被风控.
————————————
初学部署了本bot，仅在一个小群开了setu和搜图功能(非R18)，一天上线3-5个小时，
五天后被TX检测到了😢被封控了，
我的设置是开了分片，搜图、setu(发送图片非R18) 60秒自动撤回。
请问各位大佬都是如何规避TX风控的？

<img src="https://img.chkaja.com/be7c18769b794f9d.jpg" alt="be7c18769b794f9d.jpg" title="be7c18769b794f9d.jpg" />

**Comment by Tsuk1ko** [comment]
我没有什么特别的技巧

可以到 go-cqhttp 搜 issue 看看其他人的技巧

**Reply by tt9891** [reply]
感谢作者大佬的回复

**Comment by Miuzarte** [comment, marked as answer]
我以前也有段时间天天炸，挂着就完事了  迟早能解，r18x相关的功能暂时能关就关
这玩意打开也能避免部分风控
```jsonc
    // 在群组中，若结果消息有多条，采用合并转发方式发送搜图结果（频道不生效）
    "groupForwardSearchResult": true,
```
还有就是Q龄之类的也有影响，会影响被腾讯判定成是不是小号/广告号什么的
![image](https://user-images.githubusercontent.com/66856838/180178148-6984c4e8-0602-44b1-a817-65dd93ab422a.png)
以及，可以尝试一下把gocq的device.json照着真机的样子改一改`未被证实的玄学手段`
![image](https://user-images.githubusercontent.com/66856838/180178706-acc5c7e9-2d99-4686-b99f-f859e7b78d62.png)

**Reply by tt9891** [reply]
谢谢大大，已修改
